easystacks/software.eessi.io/2023.06/a64fx/eessi-2023.06-eb-5.0.0-2023a.yml

```diff
@@ -0,0 +1,9 @@
+easyconfigs:
+  - Siesta-5.2.2-foss-2023a.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/22799
+        from-commit: 0f843163c223d8c5397d1945123913108ed2e0f5
+        # Siesta testsuite fails in Deucalion's a64fx EESSI build environment
+        # but skipping the tests during build and then running them with the
+        # installed module works, so skipping the tests only in this case
+        skip-test-step: True
```

Comment:
We discussed this in a support meeting and said that given the tests do succeed when built outside of the EESSI container setup that it may well be due to FS delays in the overlays so we will move ahead with merging this.